pkg/interactor/authenticator.go

```diff
@@ -0,0 +1,16 @@
+package interactor
+
+import (
+	"errors"
+)
+
+// ErrTokenEmpty is returned for empty access tokens.
+var ErrTokenEmpty = errors.New("no access token provided")
+
+// Authenticate verifies if the access token is valid.
+func (d *DataInteractor) Authenticate(token string) error {
```

Comment:
According to what we discussed before, is it still necessary?

Reply:
I got your intention after reviewing the [other](https://github.com/CESARBR/knot-cloud-storage/pull/18/commits/2c6603c1b9148b94d3925cec5d8cf457a9aa9062) commit. However, this method is still placed as it was a use case operation. Considering our data use cases are concentrated at the `interactors/data.go` file, I think it's fine moving this method there.

Reply:
Considering the sprint deadline and that we won't be refactoring this code anytime soon... I'm moving the method there. However, I don't think it's a good practice. This could become a file with many hundreds of lines of code. It's seems a better approach to separate each use case in a separate file.

Reply:
Yes, I definitely agree that each use case should be in a separate file and that's the approach we usually follow. However, considering our reality, moving continues being the best solution to avoid the code become worse.

Reply:
Thanks!